Documentation/workflow/UsingYourBuild.md

```diff
@@ -44,22 +44,24 @@ dotnet new console
 
 ### Change project to be self-contained
 
-In order to update with your local changes the application needs to be self-contained, as opposed to running on the
+In order to update with your local changes, the application needs to be self-contained, as opposed to running on the
 shared framework.  In order to do that you will need to add a `RuntimeIdentifier` to your project.
 
-```
-  <PropertyGroup>
-    ...
-    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
-  </PropertyGroup>
+```xml
+<PropertyGroup>
+  ...
+  <RuntimeIdentifier>win-x64</RuntimeIdentifier>
+</PropertyGroup>
 ```
 
 For windows you will want `win-x64` but for other OS's you will need to set it to the most appropriate one based
```

Comment:
I think this can just tell you about `win-x64`, `osx-x64` and `linux-x64` directly.